.buildkite/basic/cloudflare-pipeline.yml

```
@@ -0,0 +1,16 @@
+steps:
+
+  - group: ":cloudflare: Cloudflare Worker Tests"
+    steps:
+    - label: ":cloudflare: Cloudflare Worker tests"
+      timeout_in_minutes: 40
+      agents:
+        queue: macos-15
```

I'd use the isolated queue as the test fixture is tied to a specific port.  This will stop two jobs (on different branches) interfering with each other.
```suggestion
        queue: macos-15-isolated
```